media/image.go

```diff
@@ -0,0 +1,17 @@
+package media
+
+import (
+	"image"
+	"image/color"
+	"image/draw"
+)
+
+func changeImageColor(src image.Image, c color.Color) image.Image {
```

Comment:
I have not tested, but I suspect this mask step isn't actually needed anymore, since the post-inkscape render will just be correctly colorized...

Reply:
Ah yeah, this is just used to colorize the fallback PNG (so the exporter works okay when inkscape isn't installed or when the media server isn't available).